hrjobcontract/hrjobcontract.php

```diff
@@ -249,7 +249,12 @@ function hrjobcontract_civicrm_pageRun($page) {
     $groups = CRM_Core_PseudoConstant::get('CRM_Core_BAO_CustomField', 'custom_group_id', array('labelColumn' => 'name'));
     $gid = array_search('HRJobContract_Summary', $groups);
 
-    CRM_Core_Resources::singleton()->addSetting(array('grID' => $gid));
+    CRM_Core_Resources::singleton()
```

Comment:
Since we import some parts of the L&A extension into JobContract this is required to make loaded Absence Type Model module work. It uses L&A settings and settings require CRM.vars to be defined.